### ISSUE TITLE
outdated manifests don't allow creation of DaemonSet

### ISSUE THREAD

**Issue by sanyer** [fluent/fluent-bit-kubernetes-logging#59]
According to kubectl [changelog for 1.15.0](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md#changelog-since-v1150):

The following APIs are no longer served by default:
* All resources under `apps/v1beta1` and `apps/v1beta2` - use `apps/v1` instead
* `daemonsets`, `deployments`, `replicasets` resources under `extensions/v1beta1` - use `apps/v1` instead
* `networkpolicies` resources under `extensions/v1beta1` - use `networking.k8s.io/v1` instead
* `podsecuritypolicies` resources under `extensions/v1beta1` - use `policy/v1beta1` instead

**Comment by andbuitra** [comment]
Indeed. This issue is affecting Kubernetes 1.15+. I will try to work on a pull request for this; it shouldn't be that hard.

**Comment by andbuitra** [comment]
I double checked and this in fact is fixed on PR https://github.com/fluent/fluent-bit-kubernetes-logging/pull/66

**Comment by gabegorelick** [comment]
I think https://github.com/fluent/fluent-bit-kubernetes-logging/pull/81 fixed most or all of these.
